.vscode/settings.json

```diff
@@ -1,6 +1,7 @@
 {
   "css.lint.validProperties": ["composes"],
   "editor.codeActionsOnSave": {
+    "source.organizeImports": false,
```

Comment:
what's the motivation for this change?

Reply:
VSCode's sorting and es-lint autofix was was breaking my changes. Every time on auto-save, vscode would override the eslint auto-fix to sort imports.

So I added this to disable the vscode import organization.

Reply:
So is this a temporary change then? Without this rule we won't have sorted imports for the future changes. Also, I think i didn't understand how exactly import sorting is breaking your changes. Could you elaborate more?

Reply:
Nope, this is a permanent fix. Sorry, let me clarify.

We have an eslint rule called [sort-imports](https://eslint.org/docs/latest/rules/sort-imports). On save, vscode automatically runs `source.fixAll.eslint` so it sorts the order, and fixes problem.

VSCode, also ships a feature with `source.organizeImports`. When I save my file after auto-sorting my imports. VSCode overrides that before saving, and uses its own formatting.

Correct:
<img width="635" alt="image" src="https://github.com/pinterest/gestalt/assets/5509813/3a148d06-ea80-4355-9bba-61edcf1d1edd">

Becomes this after save:
<img width="732" alt="image" src="https://github.com/pinterest/gestalt/assets/5509813/6631718c-ff6c-4098-96de-82c2cbd071e9">

If I don't make this change here, I can't ship my code, because my editor is broken otherwise.

Hopefully that clarifies

Reply:
Oh, now I see it. Thanks for clarifying 👍